paper-tooltip.html

```diff
@@ -106,9 +112,35 @@
         },
 
         /**
+         * Positions the tooltip to the top, right, bottom, left of its content.
+         */
+        position: {
+          type: String,
+          value: 'bottom'
+        },
+
+        /**
+         * If true, no parts of the tooltip will ever be shown offscreen.
```

Comment:
I did the renaming. I don't understand how to wrangle IronFitBehaviour for this case, but I've left a todo in the case future monica can. :(